lib/cli.js

```diff
@@ -1591,22 +1592,43 @@ CLI.prototype.do_import = function do_import(subcmd, opts, args, callback) {
         return callback(new errors.UsageError(format(
             'incorrect number of args (%d): %s', args.length, args.join(' '))));
     } else if (args.length === 1) {
-        // `*-imgadm import -S <source-url> <uuid>`
+        // `*-imgadm import -S <source-url> [-C channel] <uuid>`
         uuid = args[0];
         assert.uuid(uuid, 'uuid');
 
         if (!opts['source-url']) {
             return callback(new errors.UsageError('no source URL given'));
         }
 
+        var purl = {};
```

Comment:
It would be helpful to have a comment somewhere around here that explains that despite the help output not mentioning specifying a channel query param on the `-S source-url`, that it is supported here for backward compatibility before the `-C` option was added.